packages/sdks/snippets/angular/src/app/blog-article/blog-article.component.ts

```diff
@@ -0,0 +1,34 @@
+import { CommonModule } from '@angular/common';
+import { Component } from '@angular/core';
+import { Content, fetchOneEntry } from '@builder.io/sdk-angular';
+
+@Component({
+  selector: 'app-blog-article',
+  standalone: true,
+  imports: [Content, CommonModule],
+  template: `<div *ngIf="article" class="content">
+    <h1>{{ article.data.title }}</h1>
+    <p>{{ article.data.blurb }}</p>
+    <img [src]="article.data.image" alt="" />
+    <content [model]="model" [content]="article"></content>
+  </div> `,
+})
+export class BlogArticleComponent {
+  article: any;
+  model = 'blog-article';
+
+  constructor() {
+  }
+
+  async ngOnInit() {
+    const handle = 'new-product-line';
+    const article = await fetchOneEntry({
+      model: 'blog-article',
```

Comment:
you have a `this.model` but it's not being used here
```suggestion
      model: this.model,
```